pelican/readers.py

```diff
@@ -31,6 +31,20 @@
 # This means that _filter_discardable_metadata() must be called on processed
 # metadata dicts before use, to remove the items with the special value.
 _DISCARD = object()
+
+DUPLICATES_DEFINITIONS_ALLOWED = {
```

Comment:
At the moment, the purpose of `DUPLICATES_DEFINITIONS_ALLOWED` being a dictionary of all `False` values might be a bit puzzling. It would be valuable to add a comment here that explaining that (and perhaps also noting that `True` values might be added in the future), as that would improve future maintainability.